x-pack/plugins/reporting/server/routes/common/jobs/get_job_routes.ts

```diff
@@ -61,6 +61,14 @@ export const commonJobsRouteHandlerFactory = (reporting: ReportingCore) => {
       };
 
       if (filename) {
+        // event tracking of the downloaded file, if
+        // the report job was completed successfully
+        // and a file is available
+        const eventTracker = reporting.getEventTracker(docId, doc.jobtype, doc.payload.objectType);
+        const timeSinceCreation =
+          new Date(Date.now()).valueOf() - new Date(doc.created_at).valueOf();
```

Comment:
I haven't tested this, but look like `Date.now()` is not necessary.

```suggestion
          new Date().valueOf() - new Date(doc.created_at).valueOf();
```